### PR TITLE
Bump to 0.64 again

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -780,7 +778,7 @@
 					"$(PROJECT_DIR)",
 				);
 				LLVM_LTO = YES;
-				MARKETING_VERSION = 1.5.8;
+				MARKETING_VERSION = 1.5.6;
```
Comment (brunobar79) [inline ios/Rainbow.xcodeproj/project.pbxproj:781]: Revert

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -840,7 +838,7 @@
...
```
Comment (brunobar79) [inline ios/Rainbow.xcodeproj/project.pbxproj:841]: Same

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -936,7 +935,7 @@
...
```
Comment (brunobar79) [inline ios/Rainbow.xcodeproj/project.pbxproj:938]: Same

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1033,7 +1033,7 @@
...
```
Comment (brunobar79) [inline ios/Rainbow.xcodeproj/project.pbxproj:1036]: Same

File: src/helpers/assets.js [hunk 5]
```diff
@@ -184,7 +184,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
   const showcaseTokens = [];
   const bundledShowcaseTokens = [];
 
-  const grouped = groupBy(uniqueTokens, token => token.collection.name);
+  const grouped = groupBy(uniqueTokens, token => token.collection?.name);
```
Comment (brunobar79) [inline src/helpers/assets.js:187]: I think this was handled on a separate PR

**Review by brunobar79 — APPROVED** [review]
🔥🔥🔥

File: src/helpers/assets.js (hunk 5 above)
Comment (osdnk) [reply src/helpers/assets.js:187]: yea, sorry. Wanted to make it runnable for me

File: ios/Rainbow.xcodeproj/project.pbxproj (hunk 1 above)
Comment (osdnk) [reply ios/Rainbow.xcodeproj/project.pbxproj:781]:
```suggestion
				MARKETING_VERSION = 1.5.8;
```

File: ios/Rainbow.xcodeproj/project.pbxproj (hunk 2 above)
Comment (osdnk) [reply ios/Rainbow.xcodeproj/project.pbxproj:841]:
```suggestion
				MARKETING_VERSION = 1.5.8;
```

File: ios/Rainbow.xcodeproj/project.pbxproj (hunk 4 above)
Comment (osdnk) [reply ios/Rainbow.xcodeproj/project.pbxproj:1036]:
```suggestion
				MARKETING_VERSION = 1.5.8;
```

File: src/helpers/assets.js (hunk 5 above)
Comment (osdnk) [reply src/helpers/assets.js:187]:
```suggestion
  const grouped = groupBy(uniqueTokens, token => token.collection.name);
```

**Comment by osdnk** [comment]
@brunobar79 @skylarbarrera I find this ready to go! I fixed bugs mentioned by @skylarbarrera

**Pull request merged**